11-trt-llm/README.md

```diff
@@ -0,0 +1,75 @@
+## TRT-LLM Builder Truss
+
+### Introduction
+TRT-LLM is high-performance language model inference runtime from Nvidia. To use TRT-LLM on a model, users have to _compile_ a TRT engine from their corresponding PyTorch weights. In this Truss, we provide a config-driven interface to build the engine. We build the engine as part of the server startup and then use Triton Inference Server to serve the engine for high-performance workloads.
+
+NOTE: Building an engine can take a very long time. The approach outlined above is not scalable for fast cold-starts or production workloads where autoscaling may cause multiple replicas to build the engine repeatedly.
```

Comment:
I'm confused by this note — I thought TRT models still had a cold start time of a minute or so?